### ISSUE TITLE
Require a space between keywords and parentheses

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#170]
# Rule request

## Thesis

We should enforce this style:

```python
return (some, tuple, items)
```

Over this style:

```python
return(some, tuple, items)
```

Also fine:

```python
return some, tuple, items
```

## Reasoning

Some people use `return` and `yield` keywords as functions. The same happened to good old `print`.

We should enforce consistent parentheses usage with keywords.
I am not sure about which keywords are affected by this. One of the tasks is to try all keywords and find out which are affected.

**Comment by parejadan** [comment]
@sobolevn could you privide github param link of where it's located in the repo'e source code? Are you asking for going through the entire code base and correcting any violating code lines?

**Comment by sjortiz** [comment]
@sobolevn I can work on this if you don't mind. Thanks in advance.

**Comment by sobolevn** [comment]
@parejadan I am asking to create a new rule that checks the source code for this kind of violation. Currently we do not have it in the source code.

But, it is quite east to implement. What needs to be done?
1. Read the [API Reference](https://wemake-python-styleguide.readthedocs.io/en/latest/_pages/api.html), especially this part: https://wemake-python-styleguide.readthedocs.io/en/latest/_pages/checker.html

2. Create a new visitor (I suspect that it is going to be `tokenize` based visitor)
3. Find all keywords, take the next token, find if it is `(`
4. Check the space
5. Profit!

**Comment by valignatev** [comment]
I already did it locally, but it's blocked by #258, since adding new visitor breaks `test_all_visitors_contained_in_checker`
